_posts/2017/2017-05-09-how-to-successfully-run-a-tech-blog.md

```diff
@@ -0,0 +1,94 @@
+
+## Sharing is caring: the Twitter magic
+
+Every blog post we publish, we also share on our [ePagesDevs Twitter](https://twitter.com/epagesdevs) channel and encourage our colleagues and followers to engage in the sharing fun.
+For broadening our readership, we not only tweet about our latest blog post publications but also promote articles from the archives, publish our job openings or retweet other relevant content for our followers.
+Twitter is the most suitable Social Media platform when it comes to getting engaged with the developer community.
+
+Yet, you should not underestimate the additional workload that the maintenance of a Twitter account implies.
+Statistically, about [92% of companies](https://www.brandwatch.com/blog/44-twitter-stats-2016/) send out a tweet more than once a day.
+To engage with and keep our followers up-to-date, we tweet about *five times* per day via our [ePagesDevs account](https://twitter.com/epagesdevs) - and this even on the weekends.
+
+## Using Twitter tools
+
+But of course we do not work at our holy weekends {% emoji wink %}.
+So, thank god for Social Media tools!
+Most of our tweets are planned about a week ahead.
+In oder to do so we use the planning and analytics tool [buffer](https://buffer.com/app).
+When first connecting our Twitter account to buffer, it automatically calculated which time schedule would be best to maximize reaching out to and engaging with our followers.
+Buffer also analyzes which posts performed best and which did not.
+
+Tweet performance is also measured by the [Twitter Analytics](https://analytics.twitter.com) tool that comes along with setting up a Twitter account.
+Twitter Analytics helps you to see, which content resonates with our audience, understand how people interact with your tweets, and to get to know your followers as well as observe the growth of your follower base.
+
+Another tool that boosted our follower base was [IFTTT](https://ifttt.com/) with its function to set up [Twitter recipes](https://ifttt.com/twitter).
+One recipe that was especially efficient, was the one that automatically added those people tweeting about a specific hashtag to a Twitter list.
+Hence, these added people took notice of our account and were more likely to follow us.
+Unfortunately, this function was removed by IFTTT as it caused some errors and problems.
+Eventually, IFTTT will fix the bug and get the feature back into its repertoire.
+
+## What are you waiting for?
+
+A company's tech blog is a great opportunity to carry your development projects out to the public.
+Also, potential job candidates can check out what you are working on and thus your company tech blog will be the perfect employer branding instrument.
+And do not forget about Twitter:
+Most developers are very active on Twitter!
+Thus, it is basically a prerequisite in order to engage with the tech community.
```

Comment:
Maybe add a finishing line: "And what is your social media experience?" or "Have fun blogging!" oder "Happy Social Media"...